Makefile

```diff
@@ -60,6 +60,7 @@ integration-test:
 
 
 archive:
+	pod install
```

Comment:
Add a separate make target `pod-install` and let archive depend on it. 
```
pod-install: 
      pod install

archive: pod-install
      ...
```

Reply:
Got it. Is a --repo-update required?

Reply:
Yes. We can add it so that pod install does not fail if the spec repo has not been updated.